prompb/remote.proto

```diff
@@ -24,6 +24,12 @@ message WriteRequest {
 
 message ReadRequest {
   repeated Query queries = 1;
+
+  enum Response {
```

Comment:
Potentially we could move to `Accept` header negotiation, but it will introduce much more code to ensure backward/forwad compatibility. We have that for free in protobuf instead.

Reply:
We use the headers for versions right now (https://github.com/prometheus/prometheus/blob/master/storage/remote/client.go#L87), so I'd prefer to stick to that for consistency - for instance, we do the correct handling per version in Cortex: https://github.com/weaveworks/cortex/blob/master/pkg/distributor/http_server.go#L17

Reply:
Can agree, but it won't be that verbose ):